intelmq/bin/intelmqctl.py

```diff
@@ -1448,32 +1441,6 @@ def check(self, no_connections=False):
                 message = "Bot %r has invalid `run_mode` %r. Must be 'continuous' or 'scheduled'."
                 check_logger.warning(message, bot_id, bot_config['run_mode'])
                 retval = 1
-            if bot_id not in files[PIPELINE_CONF_FILE] and bot_config.get('enabled', True):
-                check_logger.error('Misconfiguration: No pipeline configuration found for %r.', bot_id)
-                retval = 1
-            elif bot_id not in files[PIPELINE_CONF_FILE] and not bot_config.get('enabled', True):
-                check_logger.warning('Misconfiguration: No pipeline configuration found for %r.', bot_id)
-            elif bot_id in files[PIPELINE_CONF_FILE]:
-                if ('group' in bot_config and
-                        bot_config['group'] in ['Collector', 'Parser', 'Expert']):
-                    if ('destination-queues' not in files[PIPELINE_CONF_FILE][bot_id] or
-                            (isinstance(files[PIPELINE_CONF_FILE][bot_id]['destination-queues'], list) and
-                             len(files[PIPELINE_CONF_FILE][bot_id]['destination-queues']) < 1) or
-                            (isinstance(files[PIPELINE_CONF_FILE][bot_id]['destination-queues'], dict) and
-                             '_default' not in files[PIPELINE_CONF_FILE][bot_id]['destination-queues'])):
```

Comment:
The check for destination queues should remain

Reply:
I have readded the code and also the checks for the `source_queues` as those are then used to generate a list of orphaned queues